hack/gke/xfs-formatter/xfs-formatter.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -euExo pipefail
+shopt -s inherit_errexit
+
+ephemeral_mountpoint='/mnt/stateful_partition/kube-ephemeral-ssd'
+
+raid_device="$( head -n 1 "${RAID_DEVICE_FILE}" )"
+
+systemctl stop kube-node-configuration.service
+systemctl stop kubelet.service
+crictl_pods="$( /home/kubernetes/bin/crictl pods -q )"
+/home/kubernetes/bin/crictl stopp ${crictl_pods}
+systemctl stop containerd.service
+
+tmp_dir="$( mktemp -d )"
+cp -rf "${ephemeral_mountpoint}"/. "${tmp_dir}"/
+
+function safe_umount_multiple_mount_points {
+  for line in ${1}; do
+    if mountpoint -q "${line}"; then umount "${line}"; fi
```

Comment:
nit: dedicated lines

Reply:
does it refer to formatting or explicitly running the command on each mount point (instead of having this function)?

Reply:
formatting